pydicom/encaps.py

```diff
@@ -370,3 +379,207 @@ def read_item(fp):
 
     item_data = fp.read(length)
     return item_data
+
+
+# Functions for encapsulating data
+def fragment_frame(frame, no_fragments=1):
+    """Yield one or more fragments from `frame`.
+
+    Parameters
+    ----------
+    frame : bytes
+        The data to fragment.
+    no_fragments : int, optional
+        The number of fragments (default 1).
+
+    Yields
+    ------
+    bytes
+        The fragmented data, with all fragments as an even number of bytes
+        greater than or equal to two.
+
+    Notes
+    -----
+
+    * All items containing an encoded fragment shall be made of an even number
+      of bytes greater than or equal to two.
+    * The last fragment of a frame may be padded, if necessary to meet the
+      sequence item format requirements of the DICOM Standard.
+    * Any necessary padding may be appended after the end of image marker.
+    * Encapsulated Pixel Data has the Value Representation OB.
+    * Values with a VR of OB shall be padded with a single trailing NULL byte
+      value (0x00) to achieve even length.
+
+    References
+    ----------
+    DICOM Standard, Part 5, Section 6.2 and Annex A.4
+    """
+    frame_length = len(frame)
+    if no_fragments > frame_length:
+        raise ValueError('The number of fragments is larger than the '
+                         'number of bytes in the frame.')
+
+    # We use floor to ensure we don't end up with too many fragments
+    length = int(floor(frame_length / no_fragments))
+
+    # Each item shall be an even number of bytes
+    if length % 2:
+        length += 1
+
+    offset = 0
+    for ii in range(no_fragments):
+        if ii < (no_fragments - 1):
+            # 1st to (N-1)th fragment
+            fragment = frame[offset:offset + length]
+        else:
+            # Nth fragment
+            fragment = bytearray(frame[offset:])
+
+            # Pad last fragment if needed to make it even
+            if (frame_length - offset) % 2:
+                fragment.extend(b'\x00')
+
+        offset += length
+        yield bytes(fragment)
+
+
+def itemise_fragment(fragment):
+    """Return an itemised `fragment`.
```

Comment:
Maybe `fragment_item` or `fragment_item_bytes`? Or something better sounding - I'm not good with names... This would also avoid the need for i18n ;)

Reply:
`fragment_item` sounds like you're fragmenting an item, whereas the fragment is being turned into an item. I couldn't come up with anything descriptive that avoids the s/z.